### ISSUE TITLE
Does the WAF rule count rather than block suspicious requests?

### ISSUE THREAD

**Issue by frek818** [OllieJC/aws-log4j-mitigations#3]
Does this waf rule only "count" rather than "block" when the rule matches. I'm not super familiar with waf but that's what it seems like here:

https://github.com/OllieJC/aws-log4j-mitigations/blob/bce3ef3716dbd2a45e0b82d27bc13107fb8aa104/aws_waf/rule.tf#L15

**Comment by frek818** [comment]
I came across this, which suggests the the override action be set to:

```
override_action {
    none {}
}
```

I've submitted a PR for the change but feel free do your own investigation as I don't have much personal experience with configuring WAF rules.

**Comment by frek818** [comment]
I found this while researching this issue and it seems to confirm that that setting the `override_action` to `count` will cause the rule to merely count the event rather than block it. The article suggest setting the `override_action` to `none` instead.

https://github.com/bridgecrewio/checkov/issues/2101

I've submitted a [PR](https://github.com/OllieJC/aws-log4j-mitigations/pull/4) to make that change. Feel free to double check my research because I don't have a ton of experience with waf rules.
